packages/core/scss/components/_checkboxes.scss

```diff
@@ -23,6 +23,7 @@
     min-width: 0px;
     overflow: hidden;
     position: absolute;
+    display: none;
```

Comment:
@ridvandmrc could you please take anther look at my comment so we can finalise the PR?

Reply:
After additional research we realized that both versions will remove the element from the a11y tree. This could be a problem for users relying on screen reader technology.